src/Microsoft.DotNet.Configurer/NuGetCacheSentinel.cs

```diff
@@ -23,7 +23,7 @@ private string NuGetCachePath
             {
                 if (string.IsNullOrEmpty(_nugetCachePath))
                 {
-                    _nugetCachePath = PackageDependencyProvider.ResolvePackagesPath(null, null);
+                    _nugetCachePath = NuGetPathContext.Create(new NullSettings()).UserPackageFolder;
```

Comment:
Actually there's some weird scenarios here because what if you did do `dotnet restore ./dir/project.json` and in `dir` you had a NuGet.config with `globalPackagesFolder` 

Now offline didn't save you anything. I guess it's not on the main path of this.